.github/workflows/pipeline_v3_ci.yml

```
@@ -0,0 +1,264 @@
+
+env:
+  PYTHON_VERSION: '3.12'
+  UV_VERSION: '0.5.13'
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+
+    steps:
+    - uses: actions/checkout@v4
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 6 but found 4

(indentation)